example-app/src/main/res/values-night/styles.xml

```diff
@@ -8,7 +8,7 @@
 
 <resources>
 
-    <style name="GooglePayButton">
-        <item name="buttonTheme">light</item>
+    <style name="AdyenCheckout.GooglePay.Button">
```

Comment:
Do we/merchants still need to do this manually?

Reply:
We now automatically do this